packaging/docker/README.md

```diff
@@ -403,19 +403,19 @@ proxy.
 ```yaml
 version: '3'
 volumes:
-  caddy:
+  caddy_data:
+  caddy_config:
 
 services:
   caddy:
-    image: abiosoft/caddy
+    image: caddy:<version>
```

Comment:
How about using `2` tag? I see they [have it](https://hub.docker.com/layers/caddy/library/caddy/2/images/sha256-7da0f90273e1961d9c38d26809f84d4ef3cdc9b4fc330a9cab22015d7c9e8228?context=explore).

Reply:
I think using `2` as the default makes sense. You can add a link to https://hub.docker.com/_/caddy and advice checking it if they want to change the tag.

Reply:
That's alright. I set the `2` tag.

Reply:
Thanks!